src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -0,0 +1,619 @@
+
+* Standard (head-based sampling): Before any traces arrive, we determine a set percentage of traces to accept and analyze. This gives you a solid starting point to see how tracing can help you. It is turned on by default in PHP agents 9.21.0 and higher.
+
+* Infinite Tracing (tail-based sampling): Our cloud-based service accepts all your traces and then sorts through them to find the most important. After you set up standard tracing, we recommend you add this option because it analyzes all of your traces and gives you configuration options to sample the traces that matter most to you.
+
+All the steps you need to get started with distributed tracing are right here:
+
+* [Set up tracing for new agents:](#new-agents) Steps for head- and tail-based sampling for new agent installations
+* [Older PHP agents:](#older-agents) Tracing options if you have older PHP agents
+* [Manual instrumentation:](#manual-instrumentation) Tips if automatic instrumentation doesn't work
+
+## Set up tracing for new agents [#new-agents]
+
+Whether you just want to try out standard distributed tracing (head-based sampling) or also want to set up Infinite Tracing (tail-based sampling), you need to start by setting up standard distributed tracing. We'll get you through the APM agent installation to get head-based sampling going. Then, if you want to try out Infinite Tracing, you can take some additional steps.
+
+### Install an agent to get standard distributed tracing [#quick-start-apm]
+
+This is the best approach to set up [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) if you haven't installed any APM agents for your services yet, or if you want to instrument additional services.
+
+If you already have some services instrumented with this APM agent, and you want to include them in distributed tracing, you'll need to manually enable distributed tracing for each service. See [Options for older APM agents](#older-agents).
+
+<Callout variant="tip">
+  You'll need a New Relic account to set up distributed tracing. If you don't already have one, you can quickly [create a free account](https://newrelic.com/signup).
+</Callout>
+
+#### Step 1. Identify services [#identify]
+
+Figure out which services touch your request so you can instrument each of them to send trace data to New Relic.
+
+#### Step 2. Instrument each service with an APM agent [#update-agents]
+
+You'll repeat the agent installation routine for each service involved in your transactions. If some of your services use other languages, simply repeat the [installation steps](/docs/distributed-tracing/enable-configure/quick-start) for those languages.
+
+To start the installation routine, click the tile below. When you are finished installing each agent, return here to see tips for [viewing your traces](#view-traces).
```

Comment:
general critique for all related wording: it seems weird to me to say 'click the tile below' when we could just link from the words. The tiles make sense to me (like clamshells) when there is more than one option, but seems strange when there's just a single option. it's basically just working like a big button at that point and not seeing much value there.

Reply:
In fact, i'd prob prefer a big button that read 'Start install'. but in any case, it's not a big deal and is likely fine if it draws a little attention to it.